storage/src/backend/connection.rs

```diff
@@ -363,7 +348,55 @@ impl Connection {
                 });
             }
         }
-        // TODO: check mirrors' health
+        // Check mirrors' health
+        for mirror in connection.mirrors.iter() {
+            let conn = connection.clone();
+            let connect_timeout = config.connect_timeout;
```

Comment:
Should use `config.timeout` here?

Reply:
I refer to the `proxy` health checking code. What is the difference between `connect_timeout` and `timeout`?

Reply:
I understand. It is related to the next comment.